### PR TITLE
Read permissions for token from json file

### PR DESCRIPTION
**Pull request by wencke-lm**
I would suggest reading the permissions from a file, like we do with layouts.

I also adapted some of the comments that weren't correct any longer due to an earlier pull request.

r? @TimDiekmann

### CODE REVIEW

File: scripts/create_room_token.sh [hunk 1]
```diff
@@ -6,21 +6,23 @@ set -eu
...
+#   SLURK_PORT: Port to use for the request, defaults to 5000
 
 TOKEN=${SLURK_TOKEN:=00000000-0000-0000-0000-000000000000}
 HOST=${SLURK_HOST:-http://localhost}
 PORT=${SLURK_PORT:-5000}
 
-if [ "$#" -lt 1 ]; then
-  echo "Usage: $0 room_id [registrations_left]" 1>&2
+if [ "$#" -lt 2 ]; then
+  echo "Usage: $0 room_id permissions_path [registrations_left] [task_id]" 1>&2
```
Comment (TimDiekmann) [inline scripts/create_room_token.sh:16]: Could you adjust this in the top of the file as well?

**Review by TimDiekmann — CHANGES_REQUESTED** [review]
Good Idea, otherwise, we'll end up back where we were before. 😄

**Comment by wencke-lm** [comment]
Also added a script to create a user. As all bots need to be passed a user id now. I think this script would be useful.

**Comment by TimDiekmann** [comment]
Maybe we also want to add an example directory, where the default layout and the basic permissions are stored? That way, we don't need to create files in the docs.

**Comment by wencke-lm** [comment]
Well, we have the directory slurk/slurk/views/static/layouts/ with our default layout.
I also thought about adding a permissions file to each sub directory (=each bot) in slurk-bots. This file would contain the permissions that the bot needs to run. Maybe one could use these files in the docs?

Or one renames slurk/slurk/views/static/layouts/ to slurk/slurk/views/static/defaults/ and adds the permissions file with the basic user permissions. I would be fine with that. It has the benefits that one does not need to introduce the second repository that early on in the docs. As long has we don't have the default layout in two positions.

**Comment by TimDiekmann** [comment]
The default layout isn't used anywhere in the code. The same applies to `Layout.from_json` and `Layout.from_json_file`. Thus, the name is misleading. I'd just remove the default layout from that location.

I propose to remove `Layout.from_json`, `Layout.from_json_file` and rename `Layout.from_json_data` to `Layout.from_json`. Then add/move _slurk/views/static/layouts/default.json_ to _examples/simple_layout.json_ or _scripts/example_layout.json_ and also add a simple permissions file to that location.

**Comment by wencke-lm** [comment]
Ok, sounds reasonable 👍  Between the two choices *examples/simple_layout.json* or *scripts/example_layout.json* I would prefer the first. I wouldn't expect examples in a scripts folder.

**Pull request merged**